Java/examples/src/main/java/com/amazon/randomcutforest/examples/serialization/JsonExample.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.randomcutforest.examples.serialization;
+
+import com.amazon.randomcutforest.RandomCutForest;
+import com.amazon.randomcutforest.config.Precision;
+import com.amazon.randomcutforest.examples.Example;
+import com.amazon.randomcutforest.state.RandomCutForestMapper;
+import com.amazon.randomcutforest.state.RandomCutForestState;
+import com.amazon.randomcutforest.testutils.NormalMixtureTestData;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Serialize a Random Cut Forest to JSON using
+ * <a href="https://github.com/FasterXML/jackson">Jackson</a>.
+ */
+public class JsonExample implements Example {
+    @Override
+    public String command() {
+        return "json";
+    }
+
+    @Override
+    public String description() {
+        return "serialize a Random Cut Forest as a JSON string";
+    }
+
+    @Override
+    public void run() throws Exception {
```

Comment:
Can this be run in IDE?

Reply:
I added a main method to each example so that the example code can be easily invoked from an IDE.